### DISCUSSION TITLE
vuetify template

### DISCUSSION THREAD

**Discussion by open768** [Deluze/electron-vue-template#40] [General]
i tried to add vuetify to this template can got tied up in errors as vuetify needs vue2
there is however a great vue, vuetify, electron scaffold that works first time at https://vutron.jooy2.com/getting-started.html.

**Comment by Deluze** [comment]
A new version of Vuetify recently came out that supports Vue3, check it out here: https://next.vuetifyjs.com/en/

Seems like you tried using Vuetify 2?
